.gitignore

```diff
@@ -1,10 +1,139 @@
-.idea/
```

Comment:
I'd prefer to keep this, and not the individual .idea ignores below. We don't want any IntelliJ specific stuff in this repo.